gr-filter/python/filter/file_taps_loader.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright 2018 Free Software Foundation, Inc.
+#
+# This file is part of GNU Radio
+#
+# GNU Radio is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 3, or (at your option)
+# any later version.
+#
+# GNU Radio is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with GNU Radio; see the file COPYING.  If not, write to
+# the Free Software Foundation, Inc., 51 Franklin Street,
+# Boston, MA 02110-1301, USA.
+#
+
+import numpy
+from gnuradio import gr
+from pathlib import Path
+import numpy as np
+import csv
+import sys
+
+class file_taps_loader(gr.basic_block):
+    """
+    Block to make filter taps created by the filter design tool available in grc.
+    """
+    def __init__(self, fpath, verbose):
+        gr.basic_block.__init__(self,
+            name="file_taps_loader",
+            in_sig=[],
+            out_sig=[])
+        self.taps = []
+        self.params = []
+        self.fpath = fpath
+        self.verbose = verbose
+        # Load taps
+        self.taps_from_design_tool(fpath)
+
+    def taps_from_design_tool(self, fpath):
```

Comment:
There is logic in filter_design.py open_dialog() you could reuse, and it handles float vs. complex.

Reply:
Cool! It makes sense to look at that implementation, yes. I adopted that code a bit, but it is principally the same, which enables to load something like 0.012, -0.3, 0.013  ... and (0.0012+0.14j), (0.0032-0.15j) ... without all the meta data.